src/components/Output/style.scss

```diff
@@ -137,7 +137,7 @@
   flex-shrink: 0;
   // This fixes a severe painting bug in Chrome.
   // We should try to remove this once the issue is fixed.
-  // https://bugs.chromium.org/p/chromium/issues/detail?id=870222#c10
+  // https://bugs.chromium.org/p/chromium/issues/detail?id=870222#c10;
```

Comment:
What is this change for?

Reply:
This semicolon ensures the the typed css module loader doesn't error out on the css to typescript conversion. Otherwise, it complains for lack of delimiter at the end.

Reply:
Whoa, that's a pretty weird bug. We'd have to at least document this, as otherwise it looks like a mistake.